### PR TITLE
Mobile Version Skeleton 

### PR DESCRIPTION
**Pull request by LokurasRlz**
Edit: I have fixed the things of my two last revisions.

This was my second project at Microverse. Here I had some troubles using vscode studio and GitHub but I think I was able to handle them. I have been working and giving my best all day long since I woke up.
For this project I had to set up a mobile version skeleton for my Portfolio using FlexBox from scratch, I am a beginner so it was a challenge for me but I am excited to continue learning! I hope that everything works fine, if not I will be waiting for your review to fix it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,12 @@
...
     <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <title>Portfolio Project</title>
+  
 </head>
 <body>
     <nav class="nav-bar">
```
Comment (Ismailco) [inline index.html]:
```
  <nav class="nav-bar">
        <h1 class="title">Matias Aguirre</h1>
        <a class="icon">
            <i class="fa fa-bars"></i>
        </a>
    </nav>
```
- [ ] The container of this section should be a header tag, kindly implement this so the code is more organized

Comment (Ismailco) [inline index.html]:
```
<h1 class="title">Matias Aguirre</h1>
```
- [ ] The logo should have a link (`a` tag), kindly fix this so it's a link

Comment (Ismailco) [inline index.html]:
```
<a class="icon">
            <i class="fa fa-bars"></i>
        </a>
```
- [ ] The hamburger menu should have a `nav` tag, kindly implement this it's a requirement

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+
+
+**For this project, we had to get familiar with FlexBox to make it. It was very challenging to me because I am a beginner but I am giving my best!** 
+
+
+
+
+
+## I hope that it fulfill your espectations.
```
Comment (Ismailco) [inline readme.md:53]: - [ ] [OPTIONAL] Kindly consider adding a `Get start` section with the instruction on how to use the App/Project

File: index.html [hunk 3]
```diff
@@ -37,4 +41,4 @@ <h2 class="description">I'm a Software Developer</h2>
     </section>
```
Comment (Ismailco) [inline index.html:37]:
```
<h2 class="name">Hey there. <br> I'm Matias</h2>
```
- [ ] The main title should be an `h1` tag, kindly consider changing the `h2` so you project is more close to complete

Comment (Ismailco) [inline index.html:37]:
```
<div class="images">
            <img src="images/Twitter.png" alt="Twitter img">
            <img src="images/Linkedin.png" alt="Linkedin img">
            <img src="images/medium.png" alt="Medium img">
            <img src="images/github.png" alt="Github img">
            <img src="images/angelist.png" alt="Angellist img">
        </div>
```
- [ ] The list of icons should be wrapped in a `ul` tag to group them, and `li` per icon; each icon has a link (a tag), kindly consider implementing this so your icons are organized and used as a link for external websites

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @LokurasRlz, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -7,34 +7,34 @@
...
+            <a class="icon"><i class="fa fa-bars"></i></a>
+        </nav>
+    </header>
+        <section class="headline">
 
         <div class="page">
-            <h2 class="name">Hey there. <br> I'm Matias</h2>
-            <h2 class="description">I'm a Software Developer</h2>
+            <h1 class="name">Hey there. <br> I'm Matias</h1>
+            <h1 class="description">I'm a Software Developer</h1>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Kindly make the second title `I'm a Software Developer` in an `h2` tag.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @LokurasRlz , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @LokurasRlz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Good implementation of changes required :heavy_check_mark: 
- [ ] No linters errors :heavy_check_mark: 
- [ ] Descriptive Pull Request name :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**